pyproject.toml

```diff
@@ -9,9 +9,9 @@ packages = [{include = "ray_on_golem"}]
 [tool.poetry.dependencies]
 python = "^3.8.1"
 
-ray = {version="^2.9.2", extras=["default"]}
-#golem-core = {path="../golem-core-python", develop=true}
-#golem-core = {git="https://github.com/golemfactory/golem-core-python.git", branch="main"}
+ray = {version = "^2.9.2", extras = ["default"]}
+#golem-core = {path = "../golem-core-python", develop = true}
+#golem-core = {git = "https://github.com/golemfactory/golem-core-python.git", branch = "main"}
```

Comment:
My formatter would also change `#golem-core` into `# golem-core`. What are you using?

Reply:
Well, PyCharm does not use space there. It would be great if we could use autoformat here...